client/landing/stepper/declarative-flow/tailored-ecommerce-flow.ts

```diff
@@ -106,7 +106,10 @@ export const ecommerceFlow: Flow = {
 						const returnUrl = encodeURIComponent(
 							`/setup/${ flowName }/checkPlan?theme=${
 								selectedDesign?.slug
-							}&siteSlug=${ siteSlug.replace( '.wordpress.com', '.wpcomstaging.com' ) }`
+							}&flags=signup/tailored-ecommerce&siteSlug=${ siteSlug.replace(
+								'.wordpress.com',
+								'.wpcomstaging.com'
+							) }`
 						);
```

Comment:
Non-blocking comment: This code is getting hard to read. It might be much cleaner to clean this up via `URLSearchParams`:
```suggestion
						const urlParams = (new URLSearchParams( {
							theme: selectedDesign?.slug,
							siteSlug: siteSlug.replace( '.wordpress.com', '.wpcomstaging.com' ),
							flags: 'signup/tailored-ecommerce',
						} ).toString();
						const returnUrl = encodeURIComponent(
							`/setup/${ flowName }/checkPlan?${ urlParams }`
						);
```